From: Antony Pavlov <antonynpavlov@gmail.com>
Date: Sat, 21 Jan 2017 19:18:14 +0000
X-Dgit-Generated: 1.17.dfsg-4 52c8f1637d60a0d496571bd32f7ff7d64eeafc56
Subject: psmerge: Use ps2write.

It looks like psmerge from psutils-1.17.dfsg-3 still uses pswrite:

    $ dpkg -S /usr/bin/psmerge
    psutils: /usr/bin/psmerge
    $ dpkg -s psutils | grep '^Version:'
    Version: 1.17.dfsg-3
    $ grep pswrite /usr/bin/psmerge
       exec +(qw(gs -q -dNOPAUSE -dBATCH -sDEVICE=pswrite),

Changing "pswrite" to "ps2write" fixes the 'Unknown device: pswrite'
problem, e\ .g.

Closes:#797528.

Reported-by: Kai-Martin Knaak <knaak@iqo.uni-hannover.de>

---

--- psutils-1.17.dfsg.orig/psmerge.pl
+++ psutils-1.17.dfsg/psmerge.pl
@@ -28,7 +28,7 @@ if (defined $gs)
 {
    # Just invoke gs
    $outfile = '/dev/stdout' unless defined $outfile;
-   exec +(qw(gs -q -dNOPAUSE -dBATCH -sDEVICE=pswrite),
+   exec +(qw(gs -q -dNOPAUSE -dBATCH -sDEVICE=ps2write),
 	  "-sOutputFile=$outfile", '-f', @ARGV);
    die "$prog: exec /usr/bin/gs failed\n";
 }
